### PR TITLE
Release notes for 2.2.0-rc4

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Comment by upadhyeammit** [comment]
@tbrisker updated the PR with latest changes.

File: _includes/manuals/2.2/1.2_release_notes.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Work has been started on creating a new host detail page. The initial implementa
 * graphql: add queries and mutations for settings ([#28920](https://projects.theforeman.org/issues/28920))
 
 #### Foreman - Authentication
+* When using puma with foreman 2.1 freeipa external authentication does not work ([#30535](https://projects.theforeman.org/issues/30535))
```
Comment (ekohl) [inline _includes/manuals/2.2/1.2_release_notes.md]: I updated the capitalization in Redmine. Will also be updated when we go GA so up to you if you fix it now.

**Review by ekohl — APPROVED** [review]

File: _includes/manuals/2.2/1.2_release_notes.md (hunk 1 above)
Comment (upadhyeammit) [reply _includes/manuals/2.2/1.2_release_notes.md]: Updated !

**Pull request merged**